cypress/component/admin/SortIcon.cy.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import SortIcon from "@/components/admin/dashboards/SortIcon";
+
+describe("SortIcon", () => {
+  it("Exists", () => {
+    const Parent = () => {
+      const [headers, setHeaders] = useState([{ text: "name", sort: "off" }]);
+      const [currentSort, setCurrentSort] = useState(null);
```

Comment:
this should never be null if I am not wrong, check the codebase for what the value usually is